nautobot/docs/models/extras/dynamicgroup.md

```diff
@@ -1,7 +1,166 @@
+Any filters provided by the child groups are used to filter the members from the parent group using one of three operators: **Restrict (AND)**, **Include (OR)**, or **Exclude (NOT)**. This allows for logical parenthetical grouping of nested groups by the operator you choose for that child group association to the parent.
+
+We have attempted to simplify working with these operators by giving them both human-readable and Boolean name mappings. They are as follows:
+
+- **Restrict (Boolean `AND`)** - The **Restrict** operator performs a set _intersection_ on the queryset, and is equivalent to a Boolean `AND`. Any objects matching the child filter are _restricted_ (aka _intersected_) with the preceding filter. All filter criteria must match between the filters for a member object to be included in the resultant filter.
+- **Include (Boolean `OR`)** - The **Include** operator performs a set _union_ on the queryset, and is equivalent to a Boolean `OR`. Any objects matching the child filter are _included_ (aka _unioned_) with the preceding filter. Any filter criteria may match between the filters for member objects to be included in the resultant filter.
+- **Exclude (Boolean `NOT`)** - The **Exclude** operator performs a set _difference_ on the queryset, and is equivalent to a Boolean `NOT`. Any objects matching the child filter are _excluded_ (aka _differenced_) from the preceding filter. Any matching objects from the child filter will be negated from the members of the resultant filter.
+
+Using the example group hiearchy from above, let's apply operators and explain how it would work:
+
+```no-highlight
+parent
+- first-child {weight: 10, operator: intersection}
+- second-child {weight: 20, operator: union}
+- third-child {weight: 30, operator: difference}
+  - nested-child {weight: 10, operator: intersection}
+```
+
+## Creating Dynamic Groups
+
+Dynamic Groups can be created through the UI under _Organization > Dynamic Groups_ and clicking the "Add" button, or through the REST API.
+
+Each Dynamic Group must have a human-readable **Name** string, e.g. `device-site-ams01` and a **Slug**, which should be a simple database-friendly string. By default, the slug will be automatically generated from the name, however you may customize it if you like. You may also assign a corresponding human-friendly **Description** (e.g. "Devices in site AMS01"). Finally, you must select a **Content Type** for the group that determines the filtering parameters available include objects as member into the group.
+
+Once a new Dynamic Group is created, the **Filter Fields** or **Child Groups** may be specified.
+
+!!! note
+    The content type of a Dynamic Group cannot be modified once created, so take care in selecting this initially. This helps to reduce the possibility of inconsistent data and enforces the importance of thinking about the network data model when defining a new Dynamic Group.
```

Comment:
```suggestion
    The content type of a Dynamic Group cannot be modified once created, so take care in selecting this initially. This is intended to prevent the possibility of inconsistent data and enforces the importance of thinking about the data model when defining a new Dynamic Group.
```